qubesbuilder/plugins/source_rpm/dnf/bootstrap-dnf-centos-stream.conf

```
@@ -0,0 +1,69 @@
+[main]
+cachedir=/var/cache/yum/$basearch/$releasever
+keepcache=0
+debuglevel=2
+logfile=/var/log/yum.log
+exactarch=1
+obsoletes=1
+gpgcheck=1
+plugins=1
+installonly_limit=3
+color=never
+
+reposdir=/var/empty
+
+#  This is the default, if you make this bigger yum won't see if the metadata
+# is newer on the remote and so you'll "gain" the bandwidth of not having to
+# download the new metadata and "pay" for it by yum not having correct
+# information.
+#  It is esp. important, to have correct metadata, for distributions like
+# Fedora which don't keep old packages around. If you don't like this checking
+# interrupting your command line usage, it's much better to have something
+# manually check the metadata once an hour (yum-updatesd will do this).
+# metadata_expire=90m
+
+[Stream-BaseOS]
+name=CentOS-Stream - Base
+baseurl=http://mirror.centos.org/centos/8-stream/BaseOS/$basearch/os/
+failovermethod=priority
+gpgkey=file:///usr/share/distribution-gpg-keys/centos/RPM-GPG-KEY-CentOS-Official
+gpgcheck=1
```

My understanding is that CentOS signs its metadata, so:

```suggestion
gpgcheck=1
repo_gpgcheck=1
```